paper/src/main/java/com/denizenscript/denizen/paper/utilities/PaperAPIToolsImpl.java

```diff
@@ -409,4 +409,17 @@ public void setMaterialTags(Material type, Set<NamespacedKey> tags) {
         }
         BlockTagsSetter.INSTANCE.setTags(type, tags);
     }
+
+    @Override
+    public String getCopperGolemState(CopperGolem copperGolem) {
+        return copperGolem.getWeatheringState().name();
+    }
+
+    @Override
+    public void setCopperGolemState(ElementTag variant, Mechanism mechanism, CopperGolem copperGolem) {
+        WeatheringCopperState state = Utilities.elementToRequiredEnumLike(variant, WeatheringCopperState.class, mechanism);

```

Comment:
Should be able to use the normal `Mechanism#requireEnum`/`ElementTag#asEnum`/etc. method set here, as this is just a normal enum.